src/editors/EditorContainer.tsx

```diff
@@ -119,8 +106,6 @@ export default function EditorContainer<R>({
       e.stopPropagation();
     } else if (['Enter', 'Tab', 'ArrowUp', 'ArrowDown', 'ArrowLeft', 'ArrowRight'].includes(e.key)) {
       commit();
-    } else if (e.key === 'Escape') {
```

Comment:
`InteractionsMasks` handles `keydown` as well and it seems like we can remove this event from `EditorContainer` completely in the future. `onCommitCancel` is handled by `InteractionMasks`